app-text/asciidoc/asciidoc-8.6.9-r5.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python2_7 pypy )
```

Comment:
1) Yes I removed it, due to the python-single-r1 target thing, it's absolutely safe and passes all tests
2) and yes, I should've asked @marcecj